rnn_speech_recognition/pytorch/README.md

```diff
@@ -1,44 +1,182 @@
-# DISCLAIMER
-This codebase is a work in progress. There are known and unknown bugs in the implementation, and has not been optimized in any way.
-
-MLPerf has neither finalized on a decision to add a speech recognition benchmark, nor as this implementationn/architecture as a reference implementation.
-
 # 1. Problem 
 Speech recognition accepts raw audio samples and produces a corresponding text transcription.
 
 # 2. Directions
-See https://github.com/NVIDIA/DeepLearningExamples/blob/master/PyTorch/SpeechRecognition/Jasper/README.md. This implementation shares significant code with that repository.
+
+## Steps to configure machine
+### From Docker
```

Comment:
Step 1 is missing in this list of steps. Maybe you can remove the `### From Docker` header, and have instead
`1. Clone the repository`